soc/SoC_OCV.py

```diff
@@ -25,49 +29,96 @@ def __init__(self):
 		#This is the Resistance present when the chg-discharge curve was done, not the pack resistance
 		#Should be eplicitly tested on the setup. This is a rough estimate of what I expect it was, as we never measured it.
 		IR = 0.085
-		
+
 		#read data from txt files generated during discharge test
 		with open(fName) as f_in:
-			#fill matrices with data from file
-			voltage, current, soc = np.loadtxt(itertools.islice(f_in, 1, None, 2), delimiter = '\t', usecols = (1,3,9), unpack = True)
+			#fill matrices with data from file (all are lists)
+			self.voltage, self.current, self.soc = np.loadtxt(itertools.islice(f_in, 1, None, 2), delimiter = '\t', usecols = (1,3,9), unpack = True)
 		
 		#Estimate the ocv of the cell, only taking the internal resistance into account (assuming no voltage relaxation or recovery)
-		v_ocv = voltage + IR * current
+		self.v_ocv = self.voltage + IR * self.current
+
+		self.soc = self.__selection_sort__([value for value in self.soc])
+		self.v_ocv =  self.__selection_sort__([value for value in self.v_ocv])
```

Comment:
How many points are not in order? (Is it reasonable - ie. <25, or in the hundreds?)

Reply:
Also, do we run into issues by sorting the `soc` and `v_ocv` arrays separating? Are these not aligned?